src/pages/my-buses/my-buses.html

```diff
@@ -24,7 +24,7 @@
       <span role="heading">Your Routes</span>
       <button ion-button outline item-right aria-label="Add favorite routes. Opens a popup that won't get keyboard focus, please navigate to the bottom of the page after pressing" (click)="showRouteModal()">Add Routes</button>
     </ion-item-divider>
-      <ion-item *ngIf="!routes">
+      <ion-item *ngIf="!routes || routes.length == 0">
```

Comment:
I never did that because I thought Angular recommended to do stuff like that in the component's code instead of the view.  Further research left me increasingly unsure, so I encourage you to keep an eye on my [StackOverflow question](https://stackoverflow.com/questions/44765885/lodash-in-angular-template-efficiency).